Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs

```diff
@@ -4,7 +4,116 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+            Console.WriteLine("Console Calculator Game");
+            Console.WriteLine("-----------------------");
+            Console.WriteLine("Please enter a key to continue. . .");
+            Console.ReadLine();
+            bool showMenu = true;
+            while (showMenu)
+            {
+                Menu();
+            }
+        }
+
+        public static void Menu()
+        {
+            Console.Clear();
+            Console.WriteLine("\t\t\t\t\tHere are the menus for the game: ");
+            string menu = $"\t\t\t\t\t\t 1 - Addition \n \t\t\t\t\t\t 2 - Subtraction \n \t\t\t\t\t\t 3 - Multiplication \n \t\t\t\t\t\t 4 - Division \n\n\n";
+            Console.WriteLine(menu);
+            Choice();
+        }
+
+        public static void Choice()
+        {
+            string choiceMessage = "Please input the type of game you want to play: ";
+            Console.Write(choiceMessage);
+            var choices = Console.ReadLine();
+
+            switch (choices)
+            {
+                case "1":
+                    Console.Clear();
+                    AdditionGame();
+                    break;
+                case "2":
+                    Console.Clear();
+                    SubtractionGame();
+                    break;
+                case "3":
+                    Console.Clear();
+                    MultiplicationGame();
+                    break;
+                case "4":
+                    Console.Clear();
+                    DivisionGame();
+                    break;
+                default:
+                    Console.Clear();
+                    Console.WriteLine("Out of bounds. Please try again.");
+                    Environment.Exit(0);
```

Comment:
Typing in a wrong input should not close the program, it should re-prompt the user for a valid input.